Formula/spidermonkey.rb

```diff
@@ -1,54 +1,17 @@
 class Spidermonkey < Formula
-  desc "JavaScript-C Engine"
+  desc "SpiderMonkey is Mozilla's JavaScript engine, as used in Firefox"
```

Comment:
We can use `desc "Mozilla's JavaScript engine, as used in Firefox"` not to repeat the formula’s name.